plugin/builtin/patchjson6902transformer/PatchJson6902Transformer.go

```diff
@@ -6,6 +6,7 @@ package main
 
 import (
 	"fmt"
+	"sigs.k8s.io/kustomize/kyaml/kio/kioutil"
```

Comment:
nit: Move this to line 17 with the other `sigs.k8s.io` imports (and please make sure they are in alphabetical order)